README.md

```diff
@@ -756,6 +756,8 @@ Happy Coding! -->
 </tr>
 
 <tr align="center">
+<td align="center"> <a href="https://github.com/pranjay-poddar/Dev-Geeks/tree/main/Frontend-Projects/QrComponent"> QrComponent </a></td>
```

Comment:
Why are you adding these projects?

Reply:
I am not adding these projects I don't know why it is happening

Reply:
Remove all the changes and add your project